docs/rnw-dependencies.md

```diff
@@ -27,33 +27,37 @@ Start an **elevated** PowerShell window and run:
 
 Alternatively, you can setup your environment manually:
 - Ensure Developer Mode is turned ON in Windows Settings App.
-- Install [Visual Studio 2019 (version 16.5 or greater)](https://www.visualstudio.com/downloads) **with the following options checked**:
-  - Workloads
+- It is _highly_ recommended to update the Windows system.
+- Install a recent version of [Visual Studio 2019](https://www.visualstudio.com/downloads) **with the following options checked**:
```

Comment:
This change is in response to a statement in <https://github.com/microsoft/react-native-windows/pull/7779#issuecomment-846612551>:

> [...] so we stop building with older compiler/linkers pretty soon after a VS release is marked as public. In reality we can only ever support the latest VS.

Reply:
I agree with this change. There was some recent internal discussion where we discovered VS 16.6 no longer built our master branch correctly. We don't really test older versions, so our stated support level really does have to be evergreen.

Reply:
It looks like they are updating VS again & again: https://devblogs.microsoft.com/visualstudio/visual-studio-2019-v16-10-and-v16-11-preview-1-are-available-today/